docs/src/utils/exampleKebabNameToSourceFilename.ts

```diff
@@ -0,0 +1,15 @@
+import * as _ from 'lodash/fp'
+
+/**
+ * Converts kebab-cased-example-name back into the original filename.
+ * @param {string} exampleKebabName
+ */
+const exampleKebabNameToSourceFilename = exampleKebabName => {
+  // button-example           => ButtonExample.source.json
+  // button-example-shorthand => ButtonExample.shorthand.source.json
+  return `${_.startCase(exampleKebabName)
+    .replace(/ /g, '')
```

Comment:
we can be a bit more expressive here. The only source of the whitespace could be either leading or trailing ones of the initial kebab-cased name. So, we might eliminate them before transform will happen, like that:

```js
const trimmedExampleKebabName = exampleKebabName.trim()

return `${_.startCase(trimmedExampleKebabName) ... }...`
```

While this makes code more efficient, I would put efficiency as the least worthy benefit in this case - the primary goal of this move is rather to improve explicitness of where the spaces (that regex aims to eliminate) might happen from.